cmd/thanos/store.go

```diff
@@ -206,9 +212,12 @@ func runStore(
 		return errors.Wrap(err, "create index cache")
 	}
 
+	garbageBlocksFinder := garbagecollector.NewGarbageBlocksFinder()
 	metaFetcher, err := block.NewMetaFetcher(logger, fetcherConcurrency, bkt, dataDir, extprom.WrapRegistererWithPrefix("thanos_", reg),
 		block.NewTimePartitionMetaFilter(filterConf.MinTime, filterConf.MaxTime).Filter,
 		block.NewLabelShardedMetaFilter(relabelConfig).Filter,
+		block.NewConsistencyDelayMetaFilter(logger, consistencyDelay, extprom.WrapRegistererWithPrefix("thanos_", reg)).Filter,
```

Comment:
Since you will need to touch the code again then we could move out `extprom.WrapRegistererWithPrefix` into a separate variable and use it on this line and on L216.